tests/vlan/test_vlan.py

```diff
@@ -122,34 +128,44 @@ def work_vlan_ports_list(rand_selected_dut, tbinfo, cfg_facts, ports_list, vlan_
     return work_vlan_ports_list
 
 
-def create_vlan_interfaces(work_vlan_ports_list, ptfhost):
```

Comment:
why did we remove this function? Its not clear from description on this change

Reply:
This function was used to create ptf vlan interfaces, but ptf has never used these vlan interfaces.
And this function introduces about 10 seconds delay, so router interfaces are removed in this period. But 10 seconds is not enough for all the platforms, some platform even takes 1 minute to remove router interface.
So I use below function to replace create_vlan_interfaces, and then this test will be more stable.
assert wait_until(120, 2, 0, check_portchannels_down, duthost, portchannel_interfaces), "Shutdown portchannels failed"